src/compare_covermaps.py

```diff
@@ -0,0 +1,409 @@
+import os
+from pathlib import Path
+
+import cartopy.io.shapereader as shpreader
+import ee
+import geemap
+import geopandas as gpd
+import pandas as pd
+from shapely.geometry import GeometryCollection
+from sklearn.metrics import classification_report
+
+# ee.Authenticate()
+ee.Initialize()
```

Comment:
I suggest removing both of these lines and let the user run this command from the notebook. That way it does not execute every time you import the file